Packs/qualys/Integrations/Qualysv2/Qualysv2.py

```diff
@@ -795,7 +795,9 @@
     },
     "qualys-host-list-detection": {
         # show detection score `QDS` and score contributing factors `QDS_FACTORS`
-        "api_route": API_SUFFIX + "asset/host/vm/detection/?action=list&show_qds=1&show_qds_factors=1",
+        "api_route": API_SUFFIX
```

Comment:
Please validate with Moishy that we need to change this path as well. In the ticket it was written to change only the asset fetching process....